### ISSUE TITLE
Question: How to customize SCGI port 

### ISSUE THREAD

**Issue by the-lazy-fox** [crazy-max/docker-rtorrent-rutorrent#55]
Hi @crazy-max 
I would like to run 2 of your containers next to each other behind a VPN and it seems the SCGI port start conflicting (ofc).
In your port mapping, I can see you are using the standard one 9000, is there any way to change it inside the container?
I need this because as it's behind a VPN container, it's not the outside port which cause the issue but the internal one.
Thanks for your feedback
Best
TLF

**Comment by crazy-max** [comment]
Hi @the-lazy-fox 

> and it seems the SCGI port start conflicting (ofc).
> In your port mapping, I can see you are using the standard one 9000,

It's the WebDAV port. XMLRPC port through nginx over SCGI socket is 8000.

> is there any way to change it inside the container?

We could add the ability to change this port yes.

**Comment by the-lazy-fox** [comment]
Hi @crazy-max 
Thanks for your feedback.
I wish to make more than one of them to run side by side behind a VPN and therefore be able to customize all ports to avoid conflicts.
Great if it's possible.
Thanks

**Comment by EVOTk** [comment]
Hello,
This function also interests me a lot! Thx

**Comment by crazy-max** [comment]
Should be fixed with #73

**Comment by the-lazy-fox** [comment]
Hello @crazy-max @jnsn 
I have tried on my side with this .env file 
```
RT_DHT_PORT=6881
XMLRPC_PORT=8000
RUTORRENT_PORT=33370
WEBDAV_PORT=9070
RT_INC_PORT=50000
```
And it doesn't work at all. Firefox is saying "The connection was reset" which is not the same error than while the service is not running which mean the port is there but connection get killed instantly.


Logs:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.,
[s6-init] ensuring user provided files have correct perms...exited 0.,
[fix-attrs.d] applying ownership & permissions fixes...,
[fix-attrs.d] done.,
[cont-init.d] executing container initialization scripts...,
[cont-init.d] 00-fix-logs.sh: executing... ,
[cont-init.d] 00-fix-logs.sh: exited 0.,
[cont-init.d] 01-fix-uidgid.sh: executing... ,
Switching to PGID 100...,
Switching to PUID 1026...,
[cont-init.d] 01-fix-uidgid.sh: exited 0.,
[cont-init.d] 02-fix-perms.sh: executing... ,
Fixing perms...,
[cont-init.d] 02-fix-perms.sh: exited 0.,
[cont-init.d] 03-config.sh: executing... ,
Setting timezone to Europe/Paris...,
Setting PHP-FPM configuration...,
Setting PHP INI configuration...,
Setting OpCache configuration...,
Setting Nginx configuration...,
Setting Nginx XMLRPC over SCGI configuration...,
Setting Nginx ruTorrent configuration...,
Setting Nginx WebDAV configuration...,
Initializing files and folders...,
rpc.htpasswd is empty, removing authentication...,
rutorrent.htpasswd is empty, removing authentication...,
webdav.htpasswd is empty, removing authentication...,
Checking rTorrent local configuration...,
Checking rTorrent configuration...,
Bootstrapping ruTorrent configuration...,
Removing core plugin httprpc...,
Overriding ruTorrent plugins config (create)...,
Checking ruTorrent custom plugins...,
Checking ruTorrent plugins configuration...,
  Copying autotools plugin config...,
Checking ruTorrent custom themes...,
  Copying custom theme rtModern-Plex...,
Fixing perms...,
chown: changing ownership of '/downloads/complete': Read-only file system,
[cont-init.d] 03-config.sh: exited 0.,
[cont-init.d] 04-create-services.sh: executing... ,
[cont-init.d] 04-create-services.sh: exited 0.,
[cont-init.d] ~-socklog: executing... ,
[cont-init.d] ~-socklog: exited 0.,
2021/04/12 21:47:16 [notice] 707#707: using the "epoll" event method,
[cont-init.d] done.,
[services.d] starting services,
[services.d] done.,
2021/04/12 21:47:16 [notice] 707#707: nginx/1.19.7,
2021/04/12 21:47:16 [notice] 707#707: built by gcc 9.3.0 (Alpine 9.3.0) ,
2021/04/12 21:47:16 [notice] 707#707: OS: Linux 3.10.105,
2021/04/12 21:47:16 [notice] 707#707: getrlimit(RLIMIT_NOFILE): 32000:40000,
2021/04/12 21:47:16 [notice] 707#707: start worker processes,
2021/04/12 21:47:16 [notice] 707#707: start worker process 714,
2021/04/12 21:47:16 [notice] 707#707: start worker process 715,
2021/04/12 21:47:16 [notice] 707#707: start worker process 722,
2021/04/12 21:47:16 [notice] 707#707: start worker process 734,
[12-Apr-2021 21:47:18] NOTICE: fpm is running, pid 706,
[12-Apr-2021 21:47:18] NOTICE: ready to handle connections,
```
